src/features/user/BulkCodes/forms/IssueCodesForm.tsx

```diff
@@ -242,6 +254,23 @@ const IssueCodesForm = (): ReactElement | null => {
                 value={occasion}
                 label={t('bulkCodes:occasion')}
               />
+              {bulkMethod === 'import' && (
+                <FormControl fullWidth>
+                  <InputLabel id="demo-simple-select-label">
+                    Notification language
+                  </InputLabel>
+                  <Select
+                    labelId="demo-simple-select-label"
+                    id="demo-simple-select"
+                    value={notificationLang}
+                    label="Notification language"
+                    onChange={handleLocale}
+                  >
+                    <MenuItem value={'en'}>English</MenuItem>
+                    <MenuItem value={'de'}>Deutsch</MenuItem>
```

Comment:
Text is hardcoded and not translatable - this should be addressed for all visible text, including labels, options, placeholders etc.